src/styles/sign-up-page.scss

```
@@ -226,6 +226,16 @@
       font-weight: bold;
       letter-spacing: 3px;
     }
+
+    .closeIcon {
```

```suggestion
    .close-icon {
```
Remember, all css class names should be written `in-this-format`

---

It´s done!!